frontend/app/components/address.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ComponentProps } from 'react';
+
+import clsx from 'clsx';
+
+export interface AddressProps extends ComponentProps<'address'> {
+  address: string;
+  city: string;
+  provinceState?: string;
+  postalZipCode?: string;
+  country: string;
+}
+
+export function Address(props: AddressProps) {
+  const { address, city, provinceState, postalZipCode, country, className, ...restProps } = props;
+
+  // TODO 'canada' shouldn't be hardcoded as we may deal with different values of the country field such as abbreviations
+  const isNotCanadianAddress = 'canada' !== country.toLowerCase();
+
+  return (
+    <address className={clsx('mb-0', className)} {...restProps}>
```

Comment:
Use `whitespace-pre-line` tailwind class and remove the paragraph.
```suggestion
    <address className={clsx('mb-0 whitespace-pre-line', className)} {...restProps}>
```

Reply:
Fixed. 

I had to use `whitespace-pre-wrap` to preserve the two spaces before a postal/ZIP code.